kong/conf_loader/init.lua

```diff
@@ -312,6 +312,7 @@ local CONF_INFERENCES = {
   db_cache_neg_ttl = {  typ = "number"  },
   db_resurrect_ttl = {  typ = "number"  },
   db_cache_warmup_entities = { typ = "array" },
+  normalize_req_uri = { enum = { "on", "off"} },
```

Comment:
why not a boolean?

Reply:
Replace with a new parameter and used boolean type